content/guides/core/automations/create-automations/webhook.md

```diff
@@ -0,0 +1,305 @@
+{{% /alert %}}
+
+Send a repository dispatch from W&B to trigger a GitHub action. For example, suppose you have workflow that accepts a repository dispatch as a trigger for the `on` key:
+
+```yaml
+on:
+repository_dispatch:
+  types: BUILD_AND_DEPLOY
+```
+
+The payload for the repository might look something like:
+
+```json
+{
+  "event_type": "BUILD_AND_DEPLOY",
+  "client_payload": 
+  {
+    "event_author": "${event_author}",
+    "artifact_version": "${artifact_version}",
+    "artifact_version_string": "${artifact_version_string}",
+    "artifact_collection_name": "${artifact_collection_name}",
+    "project_name": "${project_name}",
+    "entity_name": "${entity_name}"
+    }
+}
+```
+
+{{% alert %}}
+The `event_type` key in the webhook payload must match the `types` field in the GitHub workflow YAML file.
+{{% /alert %}}
+
+The contents and positioning of rendered template strings depends on the event or model version the automation is configured for. `${event_type}` will render as either `LINK_ARTIFACT` or `ADD_ARTIFACT_ALIAS`. See below for an example mapping:
+
+```text
+${event_type} --> "LINK_ARTIFACT" or "ADD_ARTIFACT_ALIAS"
+${event_author} --> "<wandb-user>"
+${artifact_version} --> "wandb-artifact://_id/QXJ0aWZhY3Q6NTE3ODg5ODg3""
+${artifact_version_string} --> "<entity>/model-registry/<registered_model_name>:<alias>"
+${artifact_collection_name} --> "<registered_model_name>"
+${project_name} --> "model-registry"
+${entity_name} --> "<entity>"
+```
+
+Use template strings to dynamically pass context from W&B to GitHub Actions and other tools. If those tools can call Python scripts, they can consume the registered model artifacts through the [W&B API]({{< relref "/guides/core/artifacts/download-and-use-an-artifact.md" >}}).
+
+- For more information about repository dispatch, see the [official documentation on the GitHub Marketplace](https://github.com/marketplace/actions/repository-dispatch).
+
+- Watch the videos [Webhook Automations for Model Evaluation](https://www.youtube.com/watch?v=7j-Mtbo-E74&ab_channel=Weights%26Biases) and [Webhook Automations for Model Deployment](https://www.youtube.com/watch?v=g5UiAFjM2nA&ab_channel=Weights%26Biases), which guide you to create automations for model evaluation and deployment. 
+
+- Review a W&B [report](https://wandb.ai/wandb/wandb-model-cicd/reports/Model-CI-CD-with-W-B--Vmlldzo0OTcwNDQw), which illustrates how to use a Github Actions webhook automation for Model CI. Check out this [GitHub repository](https://github.com/hamelsmu/wandb-modal-webhook) to learn how to create model CI with a Modal Labs webhook. 
+
+{{% /tab %}}
+
+{{% tab header="Microsoft Teams notification" value="microsoft"%}}
+
+Configure an ‘Incoming Webhook' to get the webhook URL for your Teams Channel by configuring. The following is an example payload:
```

Comment:
Incomplete sentence for: "Configure an ‘Incoming Webhook' to get the webhook URL for your Teams Channel by configuring."

Reply:
Was inherited from the previous page -- fixed!